src/Accounts/Controllers/AccountController.cs

```diff
@@ -94,10 +94,15 @@ public async Task<IActionResult> Login(LoginViewModel model, string returnUrl =
         // GET: /Account/Register
         [HttpGet]
         [AllowAnonymous]
-        public IActionResult Register(string returnUrl = null)
+        public IActionResult Register(int id = 0, string returnUrl = null)
         {
+            var model = new RegisterViewModel
+            {
+                ContactRequestId = id

```

Comment:
ContactRequestId in RegisterViewModel is no longer used since the email has a link that points directly to the request so it should be removed.

Reply:
removed